File: Level_2/올바른 괄호/solution.py
def solution(s):
    count = 0
    
    for ch in s :
        if ch == '(' :
            count += 1
        else :
            count -= 1
            
        if count < 0 :
            return False

    if count == 0 :
        return True
    else :
        return False

File: Level_2/올바른 괄호/solution.js
function solution(s){
    var answer = true;
    var count = 0;
    
    for(var i = 0; i < s.length; i++){
        var ch = s.charAt(i);
        
        if(ch == '(') count++;
        else count--;
        
        if(count < 0) return false;
    }
    
    if(count == 0) answer = true;
    else answer = false;

    return answer;
}
